content/en/blog/general/psp-migration/index.md

```diff
@@ -0,0 +1,366 @@
+---
+date: 2023-05-24
+title: PodSecurityPolicy migration with Kyverno
+linkTitle: PodSecurityPolicy migration with Kyverno
+description: Migrating from PodSecurityPolicy to Kyverno
+draft: false
+---
+
+As you've probably heard or maybe observed at this point in time, [PodSecurityPolicy](https://kubernetes.io/docs/concepts/security/pod-security-policy/) (PSP) in Kubernetes is no more. After a deprecation beginning in v1.21, they were finally removed in v1.25. Many organizations out there are still relying on PSPs and, if you're reading this post, you're probably one of them. As you begin to upgrade your clusters closer and closer towards v1.25, the clock is ticking. The choices with which you are faced are to either delay cluster upgrades, which means you aren't keeping up with the frequent releases and risk being on an unmaintained version, or upgrade to v1.25 or later and simply not have security for Pods. Neither is really an acceptable choice. In this blog post, we'll show you what your options are and provide a step-by-step migration guide for getting off PSP and onto Kyverno so you can feel confident in your ability to upgrade safely.
```

Comment:
Consider trimming / shortening sentences (e.g. remove text like "or maybe observed at this point in time,") and breaking into shorter paragraphs for easier reading.